roboflow/models/inference.py

```diff
@@ -0,0 +1,77 @@
+import io
+import urllib
+
+import requests
+from PIL import Image
+from requests_toolbelt.multipart.encoder import MultipartEncoder
+
+from roboflow.util.image_utils import validate_image_path
+from roboflow.util.prediction import PredictionGroup
+
+
+class InferenceModel:
```

Comment:
i thought it'd be cleaner in the long run to add a parent inference model that can share common functionality amongst the other inference models.

Reply:
Awesome